easybuild/easyconfigs/g/GMP/GMP-6.0.0a-intel-2015a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'GMP'
+version = '6.0.0a'
+
+homepage = 'http://gmplib.org/'
+description = """GMP is a free library for arbitrary precision arithmetic, 
+operating on signed integers, rational numbers, and floating point numbers. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'pic': True, 'precise': True}
```

Comment:
`optarch` is `True` by default, so that can't be it

I would be surprised if enabling `pic` fixes problems with precision, but it's possible.

As suggested, let's make these toolchain changes for GMP in a separate PR, and evaluate there by submitting test reports for Haswell (and other) systems.